project/src/main/scala/org/broadinstitute/clio/sbt/Dependencies.scala

```diff
@@ -105,10 +105,18 @@ object Dependencies {
 
   val UtilMainDependencies: Seq[ModuleID] = Seq(
     "com.beachape" %% "enumeratum" % EnumeratumVersion,
+    "com.beachape" %% "enumeratum-circe" % EnumeratumCirceVersion,
     "com.chuusai" %% "shapeless" % ShapelessVersion,
+    "com.fasterxml.uuid" % "java-uuid-generator" % UUIDVersion,
     "com.google.auth" % "google-auth-library-oauth2-http" % GoogleAuthHttpVersion,
     "com.iheart" %% "ficus" % FicusVersion,
     "com.typesafe" % "config" % TypesafeConfigVersion,
+    "com.typesafe.akka" %% "akka-http" % AkkaHttpVersion,
```

Comment:
Woah, why do we need akka-http in the util project?

Reply:
Ah, so we can use `OAuth2BearerToken`. I'll comment above on a way to avoid this.